package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "./node_modules/.bin/nodemon --exec 'babel-node ./src/backend/index.js'",
     "start": "babel-node ./src/backend/index.js",
-    "test": "echo \"Error: no test specified\" && exit 0"
+    "test": "echo \"Error: no test specified\" && exit 0",
```

Comment:
Why do we have this if we already `jest` set up? You can add another script for `test:backend` and reference all tests with this script as `npm run test:frontend && npm run test:backend`.